frontend/src/views/Lobby.tsx

```diff
@@ -711,6 +732,47 @@ function LobbyPage() {
                 />
               </SliderContainer>
             </HoverContainerSlider>
+            <NoMarginMediumText>Number of Problems</NoMarginMediumText>
+            <NoMarginSubtitleText>
+              {`${numProblems} problem${numProblems === 1 ? '' : 's'}`}
+            </NoMarginSubtitleText>
+            <HoverContainerSlider>
+              <HoverElementSlider
+                enabled={isHost(currentUser)}
+                onMouseEnter={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(true);
+                  }
+                }}
+                onMouseLeave={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(false);
+                  }
+                }}
+              />
+              <SliderContainer>
+                <Slider
+                  min={1}
+                  max={10}
+                  value={numProblems}
+                  disabled={!isHost(currentUser)}
+                  onChange={(e) => {
+                    const { value } = e.target;
+
+                    // Set numProblems to undefined to allow users to clear field
+                    if (!value) {
+                      setNumProblems(undefined);
+                    } else {
+                      const newNumProblems = Number(value);
+                      if (newNumProblems >= 1 && newNumProblems <= 10) {
+                        setNumProblems(newNumProblems);
+                      }
```

Comment:
I added this because all our other sliders seem to have another check, despite the number still being restricted by the slider and unallowable answers caught by the backend. I could delete this if statement, but I don't think there is much harm in leaving it in here.

Reply:
Got it - agreed, that makes sense, leave it in! 👍